docs/nodes/mamaki-testnet.md

```diff
@@ -120,12 +120,10 @@ You can return back to where you left off in the Bridge Node guide [here](./vali
 
 ## Delegate to a Validator
 
-To delegate tokens to the the `celesvaloper` validator, as an example you can run:
+To delegate tokens to the the `celestiavaloper` validator, as an example you can run:
 
-```sh
-celestia-appd tx staking delegate \
-    celesvaloper1q3v5cugc8cdpud87u4zwy0a74uxkk6u43cv6hd 1000000utia \
-    --from=$VALIDATOR_WALLET --chain-id=mamaki
+```
+celestia-appd tx staking delegate celesvaloper1q3v5cugc8cdpud87u4zwy0a74uxkk6u43cv6hd 1000000utia --from=$VALIDATOR_WALLET --chain-id=mamaki
```

Comment:
This doesn't actually look like a functional change? Revert the change, as a single long line doesn't display nicely with Docusaurus.

Reply:
The line wit the `\` also is confusing for users as there are many spaces added when copy pasting this one...!
Rather have a working line opposed to one that users fid hard to interpret...
Up to you guys!

Reply:
You can literally copy-paste the lines with `\` in them. It's shell. Please revert.

Reply:
Also, change `celesvaloper1q3v5cugc8cdpud87u4zwy0a74uxkk6u43cv6hd` to `celestiavaloper1q3v5cugc8cdpud87u4zwy0a74uxkk6u43cv6hd`